### PR TITLE
SAR-8973 - [MEI-FE] Update ReadME.md and TestREADME.md

### PR DESCRIPTION
**Pull request by robertom-binqua**

### CODE REVIEW

File: TestREADME.md [hunk 1]
```diff
@@ -62,12 +73,23 @@ This is a test entry point which simulates a service by triggering the initial P
...
+5. Business verification checkbox
+
+   - Used for skipping further verification checks carried out currently by Business Verification (SI)
+   - This is currently autofilled but can be changed
+
+6. Regime (Required)
+
+   - This is the Tax Regime Identifier
+   - It is passed down to the Registration API
+   - Accepted values are PPT or VATC
```
[the remark this answers is not in the document]
Comment (robertom-binqua) [reply TestREADME.md:85]: yes

File: TestREADME.md [hunk 2]
```diff
@@ -91,10 +113,22 @@ This is a test entry point which simulates a service by triggering the initial P
...
 
-   - Shown in the HMRC footer
+   - Used for skipping further verification checks carried out currently by Business Verification (SI)
+   - This is currently autofilled but can be changed
+
+6. Regime (Required)
+
+   - This is the Tax Regime Identifier
+   - It is passed down to the Registration API
+   - Accepted values are PPT or VATC
```
[the remark this answers is not in the document]
Comment (robertom-binqua) [reply TestREADME.md:125]: yes

File: TestREADME.md [hunk 3]
```diff
@@ -33,7 +33,18 @@ This is a test entry point which simulates a service by triggering the initial P
...
+5. Business verification checkbox 
+
+   - Used for skipping further verification checks carried out currently by Business Verification (SI)
+   - This is currently autofilled but can be changed
+
+6. Regime (Required)
+
+   - This is the Tax Regime Identifier
+   - It is passed down to the Registration API
+   - Accepted values are PPT or VATC
```
[the remark this answers is not in the document]
Comment (robertom-binqua) [reply TestREADME.md:45]: Refaque plese write down the format you prefer:

Is this correct?

6. Regime (Required)

   - This is the Tax Regime Identifier
   - It is passed down to the Registration API
   - This is currently defaulted to VATC. Accepted values are PPT or VATC

[the remark this answers is not in the document]
Comment (refaque-ahmad) [reply TestREADME.md:45]: I would write "This is currently defaulted to VATC but accepted values are PPT or VATC". What do you think mate :)

**Comment by robertom-binqua** [comment]
Updated to "This is currently defaulted to VATC but accepted values are PPT or VATC" as Refaque suggested

**Review by refaque-ahmad — APPROVED** [review]

**Pull request merged**